app/code/Magento/Checkout/view/frontend/web/js/model/place-order.js

```diff
@@ -37,7 +37,9 @@ define(
                         'newCustomerBillingAddress': null
                     };
 
-                    if (response.responseType !== 'error') {
```

Comment:
kept for compatibility reason in case some store still use old API method savePaymentInformationAndPlaceOrder